scripts/averaging/averaging_example.py

```diff
@@ -70,33 +79,19 @@ def averaging_example(case_name, input_ts_loc, output_loc, var_list, overwrite_c
         #Average time dimension over time bounds, if bounds exist:
         if 'time_bnds' in cam_ts_data:
             time = cam_ts_data['time']
-            tbnd = cam_ts_data['time_bnds'].load().mean(dim='nbnd')
-            time = xr.DataArray(tbnd.values, dims=time.dims, attrs=time.attrs)
+            time = xr.DataArray(cam_ts_data['time_bnds'].load().mean(dim='nbnd').values, dims=time.dims, attrs=time.attrs)  # NOTE: force `load` here b/c if dask & time is cftime, throws a NotImplementedError
             cam_ts_data['time'] = time
             cam_ts_data.assign_coords(time=time)
             cam_ts_data = xr.decode_cf(cam_ts_data)
-        else:
-            # rely on the time dimension being correct
-            cam_ts_data = xr.decode_cf(cam_ts_data)
 
         #Group time series values by month, and average those months together:
-        climo_data = cam_ts_data[var].groupby('time.month').mean(dim='time')
-
-        #Add surface pressure (PS) climatologies to each data set:
-        if "PS" in cam_ts_data:
-            ps_climo = cam_ts_data['PS'].groupby('time.month').mean(dim='time')
-            cam_climo_data = xr.merge([climo_data, ps_climo])
-        else:
-            cam_climo_data = climo_data.to_dataset()
-
-        #Also add some special variables that should be carried along:
-        for special in ['hyam', 'hybm', 'P0', 'hyai', 'hybi']:
-            if special in cam_ts_data:
-                cam_climo_data[special] = cam_ts_data[special]
```

Comment:
Is this code no longer needed?  I thought for spectral element CAM runs these quantities are missing unless added explicitly?

Reply:
I'm going to defer on this for now. You're probably right, but I can't remember exactly why I did this. Let's skip it for now, and if needed, that can be dealt with in a different PR.